examples/collect_data.py

```diff
@@ -7,25 +7,53 @@
 
 from gym_urbandriving.agents import AccelAgent
 
+def early_stop_actions(actions):
+    """
+    Helper function used to determine if list of actions indicates that a demonstration can be terminated early. 
+
+    Params
+    --------
+    actions : list
+        List of actions taken by all the agents in a time step. 
+
+    Returns
+    -------
+    bool
+        True if approximately all the cars have gone through the intersection and are back up to speed. 
+
+    """
+    return actions == [(0, 1), (0, 1), (0, 1), (0, 1)]
+
 def f():
```

Comment:
Can we remove the cProfile("f") call and replace with just f()? The cProfile call really just collects a lot of profiling data. Lets replace that with just f().

Reply:
I was thinking an if main would be more standard? We can change that in future versions though.

Reply:
changed to if main